cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -110,14 +113,30 @@ def upload_snapshot(manager, local_path, snapshot_id, logger):
                 json.dumps(snapshot, indent=2))
 
 
-def restore_snapshot(manager, snapshot_id, cfy, logger,
+def list_snapshots(manager, logger):
+    logger.info('Listing snapshots:')
+    snapshots = manager.client.snapshots.list()
+    for snapshot in snapshots:
+        logger.info('%(id)s - %(status)s - %(error)s', **snapshot)
+
+
+def list_executions(manager, logger):
+    logger.info('Listing executions:')
+    executions = manager.client.executions.list(include_system_workflows=True)
+    for execution in executions:
+        logger.info('%(id)s (%(workflow_id)s) - %(status_display)s',
+                    **execution)
```

Comment:
`**execution` is a TypeError here, has to be just `execution`

eg:
```python
>>> logging.info('asd %(x)s', x=5)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python2.7/logging/__init__.py", line 1629, in info
    root.info(msg, *args, **kwargs)
  File "/usr/lib/python2.7/logging/__init__.py", line 1167, in info
    self._log(INFO, msg, args, **kwargs)
TypeError: _log() got an unexpected keyword argument 'x'
>>> logging.info('asd %(x)s', {'x': 5})
INFO:root:asd 5
>>> 
```

Reply:
ditto for other usages and also `**snapshot` etc

Reply:
Huh, wonder how I missed that. Updated.